.mci.yml

```diff
@@ -2221,3 +2221,13 @@ buildvariants:
       display_name: silk
       tasks:
         - name: .silk
+
+    - name: rhel79-compile
+      display_name: "RHEL 7.9 (gcc 4.8.5)"
+      expansions:
+          build_type: "Release"
```

Comment:
Why release?

Reply:
Along with `BSONCXX_POLYFILL: impls`, I would rather test options recommended for install. Release matches the build type [shown in install instructions](https://www.mongodb.com/docs/languages/cpp/cpp-driver/current/installation/linux/#step-3--configure-the-driver).